.vscode/settings.json

```diff
@@ -13,10 +13,10 @@
         "**/systori/templates/**/*": "django-txt",
         "**/requirements{/**,*}.{txt,in}": "pip-requirements",
         "*.pip": "pip-requirements",
-        "*.pgsql": "sql"
+        "*.pgsql": "sql",
     },
     "emmet.includeLanguages": {
-        "django-html": "html"
+        "django-html": "html",
     },
     "python.linting.enabled": true,
     "python.linting.pep8Enabled": false,
```

Comment:
why?

Reply:
also, line 8 - why hide dart folder?

Reply:
It needs dart 1 and globally I have flutter (dart 2) also, it slows down vscode because dart 1 was slow and with python and docker running vscode slows down.

The `dart` folder also has a lot of build files so the indexing of multiple extensions slows vscode.

Another reason is that our folder hierarchy is very long, so it's not very easily browsable

If `dart` folder is required for local work/testing it can be added to vscode using
`File -> Add folder to workspace` without changing out setting.

This will be the result

![Workspace folders](https://user-images.githubusercontent.com/18462563/63285903-68e1e200-c2d4-11e9-98d0-8ce4c3a92ded.png)